pyproject.toml

```diff
@@ -203,6 +219,7 @@ select = ["ALL"]
     "D101",    # tests don't need docstrings
     "D103",    # tests don't need docstrings
     "PLR0915", # tests are allowed to be long, sometimes that's convenient
+    "PT004",   # Not returning anything is totally reasonable for fixtures
```

Comment:
Why not follow the advice of this diagnostic and rename them to use underscores in that case? That's what I've been doing. It's apparently a convention, and it makes sense to me.

Reply:
#1 that seems weird, and #2 that made me have to go through and annotate everydamthing with @pytest.mark.usefixtures, but oh well, done now.

Reply:
Sorry, I don't understand, why would you have to use `@pytest.mark.usefixtures`? You still use the fixture the same way as always, by making it a parameter to the test function. The point of the leading `_` is that it makes clear that parameter is not a real parameter and should not be used in the body of the function.